assets/csidriveroperators/aws-ebs/01_namespace.yaml

```diff
@@ -2,3 +2,6 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: openshift-cluster-csi-drivers
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
```

Comment:
This line is to match https://github.com/openshift/cluster-storage-operator/commit/d558dbc19a79aa2f3cc758f2d49a06faabf3d4c2

Reply:
In theory we could simply remove these files from `assets` because the `openshift-cluster-csi-drivers` namespace is created by CVO.